src/libraries/Microsoft.XmlSerializer.Generator/src/GenerateNupkgProps.targets

```diff
@@ -19,6 +19,12 @@
   &lt;ItemGroup&gt;
     &lt;DotNetCliToolReference Include="$(PackageId)" Version="$(PackageVersion)" /&gt;
   &lt;/ItemGroup&gt;
+  &lt;PropertyGroup&gt;
+    &lt;AddSerializers&gt;False&lt;/AddSerializers&gt;
```

Comment:
My gut says this should default to True? Doing so is a behavior change, but it's a behavior that people have noticed is missing and have been asking to get fixed. Perhaps others with more experience in this area have more informed opinions than my gut though?